### PR TITLE
Modifying SQL queries - a first baby step

### PR DESCRIPTION
**Pull request by agorina**
I am trying to make the SQL more manageable. 
This is the first baby step: I moved all the shared permissions queries into shared properties.

### CODE REVIEW

File: webapp/src/main/resources/application.sql.yml [hunk 1]
```diff
@@ -253,11 +182,11 @@ sql:
           from exam e
             join student_group_membership sgm on e.student_id=sgm.student_id
             join asmt a on e.asmt_id=a.id
-            join school s on e.school_id=s.id
+            join school sc on e.school_id=sc.id
           where e.school_year=:school_year
             and sgm.student_group_id=:group_id
             and (0=:group_subject_id or a.subject_id=:group_subject_id)
```
Comment (esotrope) [inline webapp/src/main/resources/application.sql.yml]: you could also factor `(0=:group_subject_id or a.subject_id=:group_subject_id)` out into `groupPermission` here

File: webapp/src/main/resources/application.sql.yml [hunk 2]
```diff
@@ -702,3 +583,63 @@ sql:
           OR (ir.org_level = 'District' AND district.id IS NOT NULL)
           OR (ir.org_level = 'SchoolGroup' AND school_group.id IS NOT NULL)
         )
+
+  reporting:
```
Comment (esotrope) [inline webapp/src/main/resources/application.sql.yml]: if we put these in `common.sql.yml` do these not come through to each app's sql yml?

**Comment by agorina** [comment]
Sorry, will reopen shortly

File: webapp/src/main/resources/application.sql.yml (hunk 1 above)
Comment (agorina) [reply webapp/src/main/resources/application.sql.yml]: Yeah, maybe later. Baby steps at this point. While these queries look very similar there are some differences and I am trying to make sure I do not break anything.

File: webapp/src/main/resources/application.sql.yml (hunk 2 above)
Comment (agorina) [reply webapp/src/main/resources/application.sql.yml]: They do, but they are not really common. They are just common for the reporting (meaning this file) so I did not see the value in putting it into `common.sql.yml`. Do you see it differently?

Comment (esotrope) [reply webapp/src/main/resources/application.sql.yml]: Ah i see. i thought there was crossover between reporting and the report processor - thanks

**Pull request closed without merging**